tests/test_event_dispatcher.py

```diff
@@ -85,3 +86,24 @@ def log_event(evt, _):
     orthanc.on_change(orthanc.ChangeType.STABLE_STUDY, orthanc.ResourceType.STUDY, 'uuid')
     assert 'change_type=STABLE_STUDY' in caplog.text
     assert 'resource_type=STUDY' in caplog.text
+
+
+def test_create_session_shall_pass_ssl_cert_if_ssl_is_enabled_and_report_issues():
+    configured_orthanc = OrthancApiHandler(
+        config={
+            'SslEnabled': True,
+            'SslCertificate': 'path-to-non-existing-file'
+        })
+    with pytest.raises(IOError, match='.*TLS CA.*invalid path: path-to-non-existing-file'):
+        event_dispatcher.create_session(configured_orthanc)
+
+
+def test_create_session_shall_not_pass_ssl_cert_if_ssl_is_disabled():
+    configured_orthanc = OrthancApiHandler(
+        config={
+            'SslEnabled': False,
+            'SslCertificate': 'path-to-non-existing-file'
+        })
+
+    client = event_dispatcher.create_session(configured_orthanc)
+    assert client is not None
```

Comment:
Does not test what the test title indicates. 

Expected something like: 
```
assert client.cert == False
```

Reply:
unfortunately there is no straightforward correspondence. When enabled, there is a private member somewhere deep down below HttpTransport / SSLContext. I'll give it a try or make more clear that no exception is raised from the creation of the client